apps/scan/backend/src/app_config.test.ts

```diff
@@ -35,8 +35,8 @@ import {
 } from '@votingworks/backend';
 import { InsertedSmartCardAuthApi } from '@votingworks/auth';
 import { CVR, ElectionDefinition } from '@votingworks/types';
-import { configureApp } from '../../../test/helpers/shared_helpers';
-import { scanBallot, withApp } from '../../../test/helpers/custom_helpers';
+import { configureApp } from '../test/helpers/shared_helpers';
+import { scanBallot, withApp } from '../test/helpers/custom_helpers';
```

Comment:
Completely unrelated. Just had a hunch that `app_config.test.ts` wasn't intentionally located inside `apps/scan/backed/src/scanners/custom/`. The other tests in `apps/scan/backed/src/scanners/custom/` are all about scanning and prefixed with `app_scan_*`. Moved it up alongside other more general tests in `apps/scan/backend/src/`.